### ISSUE TITLE
Remove trailing whitespace in tokens

### ISSUE THREAD

**Issue by BLKSerene** [Esukhia/pybo#33]
When the text to be tokenized includes non-tibetan words that use whitespace as the word boundary (for example english words), there's a trailing whitespace for each token in the results.

While it is easy to strip the trailing whitespace using `rstrip()` by users of pybo, I think that this should be made as the normal behavior?

If the tokens are to be detokenized into running text later, then a specialized detokenizer is needed to add the whitespace back only to non-tibetan words that use whitespace as the word boundary.

**Comment by drupchen** [comment]
This kind of issue is exactly why the Token objects have so many different attributes:
Check [this notebook](https://github.com/Esukhia/pybo/blob/master/Using%20BoTokenizer.ipynb)

Token.content – the unmodified content straight from the input string ("བདེ་་ལེ གས", "མཐ",  "འི་", ...)
Token.cleaned_content – the normalized form of Token.content ("བདེ་ལེགས་", "མཐ", "འི་", ...)
Token.unaffixed_word – Token.cleaned_content augmented with the འ reinsertion ( "བདེ་ལེགས་", "མཐའ་", "འི་", ...)
Token.lemma – The current word in its canonical form ("བདེ་ལེགས་", "མཐའ་", "གི་", ...)

You even have :
Token.syls – Individual syllables of every token: [[0, 1, 2], [5, 6, 8, 9]] a list of indices of non-punctuation chars for the raw substring "བདེ་་ལེ གས"
That way, you can even recreate the word and add your own punctuation should you want to do so.

In short, instead of proposing a single way of doing things that won't ever fit all needs, I prefer to give you all the ingredients to create your own recipe.

**Comment by drupchen** [comment]
once you have the list of tokens and you want to produce your output, you could put up small functions like 
```python
def extract_cleaned_content(tokens):
    out = []
    for t in tokens:
        if token.type == 'syl':
            out.append(t.cleaned_content)
        else:
            out.append(t.content)  # cleaned_content is empty for all tokens that are not tibetan text and tibetan punctuation
    return out
```

**Comment by BLKSerene** [comment]
Thanks for the information!

**Comment by C0NTRIBUTE** [comment]
FYI 

> Check this notebook

Is broken link.
